pkg/codegen/codegen.go

```diff
@@ -228,7 +228,8 @@ func Generate(swagger *openapi3.Swagger, packageName string, opts Options) (stri
 		return "", errors.Wrap(err, "error flushing output buffer")
 	}
 
-	goCode := buf.String()
+	// remove any byte-order-marks which break Go-Code
+	goCode := strings.Replace(buf.String(), "\uFEFF", "", -1)
```

Comment:
huh, weird. Could you make a function, something like "sanitizeCode" or something which will be the future place to put additional such filters? Thanks.